packages/styling/src/styles/theme.ts

```diff
@@ -101,7 +101,7 @@ function _loadFonts(theme: ITheme): { [name: string]: string } {
     for (const propName of Object.keys(font)) {
       const name = fontName + propName.charAt(0).toUpperCase() + propName.slice(1);
       let value = font[propName];
-      if (propName === 'fontSize' && value.indexOf('px') < 0) {
+      if (propName === 'fontSize' && value && value.indexOf && value.indexOf('px') < 0) {
```

Comment:
Philip & I spoke offline about using typeof to check if the value is a string instead of value.indexOf but it's just another option & this may be more elegant.